src/xmipp/applications/scripts/denoising_tv/denoising_tv.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+# *****************************************************************************
+# *
+# * Authors:     J.L. Vilas (jlvilas@cnb.csic.es) [1]
+# *
+# * [1] Centro Nacional de Biotecnologia, CSIC, Spain
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 3 of the License, or
+# * (at your option) any later version.
+# *
+# * This program is distributed in the hope that it will be useful,
+# * but WITHOUT ANY WARRANTY; without even the implied warranty of
+# * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# * GNU General Public License for more details.
+# *
+# * You should have received a copy of the GNU General Public License
+# * along with this program; if not, write to the Free Software
+# * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# * 02111-1307  USA
+# *
+# *  All comments concerning this program package may be sent to the
+# *  e-mail address 'scipion@cnb.csic.es'
+# *
+# *****************************************************************************
+
+import sys
+import os
+import mrcfile
+
+import numpy as np
+
+import scipy as sp
+import tigre
+
+import xmippLib
+from xmipp_base import XmippScript
+
+
+class DenoisingTV(XmippScript):
+    _conda_env="xtomo_tigre_denoising" 
```

Comment:
I'm not sure this env should be here if neither the plugin or xmipp generate it.

Reply:
Solved, see recent changes